pages/admin/invite/userRequest.tsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useState } from "react";
+import DeclineButton from "components/declineButton";
+import AcceptButton from "components/acceptButton";
+
+const UserRequestDashboardItem: React.FunctionComponent<UserRequest> = ({
+  name,
+  email,
+}) => {
+  return (
+    <div>
+      <div className="flex flex-row justify-between text-sm h-16 items-center">
+        <div className="flex flex-row justify-between">
+          <div className="w-10 h-10 mr-4 bg-placeholder rounded-full" />
+          <div className="w-32">
+            <p className="font-display">{name}</p>
+            <p>User Role</p>
+          </div>
+        </div>
+        <div className="w-56">
+          <p>{email}</p>
+        </div>
+        <div className="flex flex-row justify-between">
+          <DeclineButton />
+          <div className="ml-10">
+            <AcceptButton />
```

Comment:
Nit: One thing to consider is using our generic Button component, since those will have all the props you need + styling options can be applied directly to the `className` prop.